docs/content/index.html

```diff
@@ -15,7 +15,35 @@
 <head>
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width">
-
+	<link rel="schema.dcterms" href="http://purl.org/dc/terms/">
+	<link rel="schema.AGLSterms" href="http://www.agls.gov.au/agls/terms/">
+	<meta name="dcterms.title" content="Content page template - Australian Government Design System" />
+	<meta name="dcterms.creator" content="scheme=AGLSTERMS.AglsAgent; corporateName=Digital Transformation Agency, Australia; address=50 Marcus Clarke St Canberra ACT 2601; contact=+61 427 136 791" />
+	<meta name="dcterms.publisher" content="scheme=AGLSTERMS.AglsAgent; corporateName=Digital Transformation Agency, Australia; address=50 Marcus Clarke St Canberra ACT 2601; contact=+61 427 136 791" />
+	<meta name="dcterms.description" content="Content pages display information clearly" />
+	<meta name="dcterms.modified" content="scheme=dcterms.ISO8601; 2019-08-06T11:36:29+10:00" />
+	<meta name="dcterms.subject" content="design system, templates" />	
+	<meta name="dcterms.language" content="dcterms.RFC4646; en-AU" />
+	<meta name="dcterms.type" content="Content page template" />
+	<meta name="dcterms.coverage" content="Australia" />
+	<meta name="dcterms.spatial" content="Australia" />
+	<meta name="AGLSTERMS.documentType" content="text" />
+	<meta name="msapplication-TileColor" content="#ffffff">
+	<meta name="theme-color" content="#ffffff">
+	<meta name="robots" content="index, follow">
+	<meta name="author" content="Digital Transformation Agency">
+	<meta name="description" content="Content pages display information clearly">
+	<meta name="twitter:card" content="summary_large_image">
+	<meta name="twitter:title" content="Australian Government Design System">
+	<meta name="twitter:description" content="Content pages display information clearly">
+	<meta name="twitter:name" content="Australian Government Design System">
+	<meta name="twitter:image" content="https://designsystem.gov.au/assets/favicons/designsystem.jpg">
+	<meta property="og:type" content="website">
+	<meta property="og:title" content="Content page template - Australian Government Design System">
+	<meta property="og:site_name" content="Australian Government Design System">
+	<meta property="og:description" content="Content pages display information clearly">
+	<meta property="og:image" content="https://designsystem.gov.au/assets/favicons/designsystem.jpg">
```

Comment:
Consider adding:
`<meta property="og:image" content="Coat of Arms - Australian Government - Design System logo" />`

(Replicating `twitter:image:alt`)